index.bs

```diff
@@ -2047,10 +2029,6 @@ partial interface MLGraphBuilder {
     : <dfn>bias</dfn>
     ::
         An additional 1-D tensor with the shape of *[outputChannels]* whose values are to be added to the convolution result.
-
-    : <dfn>activation</dfn>
-    ::
-        An optional activation function that immediately follows the convolution operation.
```

Comment:
To be honest I don't think we should be adding notes about op fusion for any specific operators, since which operators can be fused is backend-specific. I've instead expanded on the high-level discussion of op fusion. Thoughts?

Reply:
I agreed we should put it on the high-level discussion. op fusion is platform specific, and we won't want to cover every single op.